lib/model/query/entities.js

```diff
@@ -63,8 +63,8 @@ select ins.*, def.id as "entityDefId", ds."acteeId" as "dsActeeId", ds."name" as
 
 
 // Entrypoint to where submissions (a specific version) become entities
-const _processSubmissionDef = (submissionDefId) => async ({ Datasets, Entities, Submissions }) => {
-  const existingEntity = await Entities.getDefBySubmissionDefId(submissionDefId);
```

Comment:
This was the only place where `Entities.getDefBySubmissionDefId()` was used, right? Is it worth removing it?

Reply:
I'll take it out - It's not used, but I was thinking "this feels like it may be useful some day!" If so, I guess I'll bring it back then.